test/integration/sagemaker/test_mnist.py

```diff
@@ -110,6 +112,40 @@ def test_s3_plugin(sagemaker_session, ecr_image, instance_type, region, framewor
     _assert_checkpoint_exists(region, estimator.model_dir, 200)
 
 
+def test_tuning(sagemaker_session, ecr_image, instance_type, framework_version):
+    resource_path = os.path.join(os.path.dirname(__file__), '../..', 'resources')
```

Comment:
`'..', '..'`

also, you could just add 'mnist' to this path since it's defined specifically for this test